src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonDrawer/components/AttributeInformation.tsx

```diff
@@ -117,8 +117,10 @@ const AttributeInformation = ({ selectedPolygon }: { selectedPolygon: SitePolygo
 
   useEffect(() => {
     const format =
-      calculatedArea && calculatedArea.toLocaleString("UTC", { minimumFractionDigits: 2, maximumFractionDigits: 2 });
-    setFormattedArea(format || "");
+      calculatedArea !== null && calculatedArea !== undefined
+        ? calculatedArea.toLocaleString("en-US", { minimumFractionDigits: 2, maximumFractionDigits: 2 })
+        : "";
```

Comment:
This can be simplified to `const format = calculatedArea?.toLocaleString(...) ?? ""`

Reply:
Done.

Reply:
in relation to the [comment](https://github.com/wri/wri-terramatch-website/pull/248#discussion_r1633846060):
> A GET request should not be causing any modification in the BE. It should be a POST or PATCH if it causes mutations.
The switch from GET to POST has already been implemented in both the frontend and the backend. However, for these changes to take effect, approval of this [PR in the backend](https://github.com/wri/wri-terramatch-api/pull/277) is required.